packages/react-ui-rent/src/LeadForm/Fields/OptInNewsLetter.js

```diff
@@ -1,27 +1,57 @@
 import React, { PureComponent } from 'react'
 import PropTypes from 'prop-types'
+import themed from 'react-themed'
 import { Field } from '@rentpath/react-ui-core'
 
-export default class Email extends PureComponent {
+@themed(/^OptInNewsLetter/, {
+  pure: true,
+})
+
+export default class OptInNewsLetter extends PureComponent {
   static propTypes = {
+    theme: PropTypes.object,
     name: PropTypes.string,
+    id: PropTypes.string,
     type: PropTypes.string,
     label: PropTypes.oneOfType([
       PropTypes.string,
       PropTypes.object,
       PropTypes.func,
     ]),
+    defaultChecked: PropTypes.bool,
   }
 
   static defaultProps = {
+    theme: {},
     name: 'opt_in_newsletter',
+    id: 'lead_form_opt_in_newsletter',
     type: 'checkbox',
     label: 'Simplify my search with helpful tips and rental recommendations.',
+    defaultChecked: true,
   }
 
   render() {
+    const {
+      theme,
+      label,
+      id,
+      ...props
+    } = this.props
+
     return (
-      <Field {...this.props} />
+      <div className={theme.OptInNewsLetter}>
+        <Field
+          id={id}
+          {...props}
+        />
+        {/* eslint-disable jsx-a11y/label-has-for */}
+        <label
```

Comment:
I'm not getting why you moved Label out.  Also, there is a `Label` component you should be using instead.